rotkehlchen/tests/api/test_eth2.py

```diff
@@ -335,6 +339,53 @@ def test_add_get_delete_eth2_validators(rotkehlchen_api_server, start_with_valid
     result = assert_proper_response_with_result(response)
     assert result == {'entries': [validators[1].serialize()], 'entries_limit': expected_limit, 'entries_found': 1}  # noqa: E501
 
+    # Try to add validator with a custom ownership percentage
+    custom_percentage_validtors = [
+        Eth2Validator(
+            index=5235,
+            public_key='0x827e0f30c3d34e3ee58957dd7956b0f194d64cc404fca4a7313dc1b25ac1f28dcaddf59d05fbda798fa5b894c91b84fb',  # noqa: E501
+            ownership_proportion=FVal(0.4),
+        ),
+        Eth2Validator(
+            index=43948,
+            public_key='0x922127b0722e0fca3ceeffe78a6d2f91f5b78edff42b65cce438f5430e67f389ff9f8f6a14a26ee6467051ddb1cc21eb',  # noqa: E501
+            ownership_proportion=FVal(0.5),
+        ),
+    ]
+
```

Comment:
remove extra newline here
```suggestion
    ]
```